TVHeadEnd/TVHeadEnd.csproj

```
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <AssemblyVersion>8.0.0.0</AssemblyVersion>
-    <FileVersion>8.0.0.0</FileVersion>
+    <AssemblyVersion>8.1.0.0</AssemblyVersion>
+    <FileVersion>8.1.0.0</FileVersion>
```

The Jellyfin Plugins only use the major version for their releases.
```suggestion
    <AssemblyVersion>9.0.0.0</AssemblyVersion>
    <FileVersion>9.0.0.0</FileVersion>
```

---

Version has already been bumped, but no release was cut for it.